backend/keystore/keystore.go

```diff
@@ -53,8 +53,8 @@ type Keystore interface {
 	// This is typically done through a screen on the device or through a paired mobile phone.
 	// optional is true if the user can skip verification, and false if they should be incentivized
```

Comment:
I wonder what "incentivized" means exactly. I'd keep it simpler: "optional is true if the user can skip verification." and that's it - it's either optional or non-optional.